src/Sequence.cpp

```diff
@@ -189,60 +189,18 @@ std::size_t extractPadding( const std::vector<detail::FileNumbers>::const_iterat
 	return 0;
 }
 
-
-/**
- * @brief return if the padding is strict (at least one frame begins with a '0' padding character).
- * @param[in] timesStr vector of frame numbers in string format
- * @param[in] padding previously detected padding
- */
-bool extractIsStrictPadding( const std::vector<std::string>& timesStr, const std::size_t padding )
-{
-	if( padding == 0 )
-	{
-		return false;
-	}
-
-	BOOST_FOREACH( const std::string& s, timesStr )
-	{
-		if( s[0] == '0' )
-		{
-			return true;
-		}
-	}
-	return false;
-}
-
-
-bool extractIsStrictPadding( const std::vector<detail::FileNumbers>& times, const std::size_t i, const std::size_t padding )
-{
-	if( padding == 0 )
-	{
-		return false;
-	}
-
-	BOOST_FOREACH( const detail::FileNumbers& s, times )
-	{
-		if( s.getString( i )[0] == '0' )
```

Comment:
I don't think so because _strictPadding == 0 is the same as _strictPadding == false.